test/e2e/tools/eks-anywhere-test-tool/pkg/cloudwatch/cloudwatch.go

```diff
@@ -85,3 +90,10 @@ func (c Cloudwatch) getLogSegment(logGroupName string, logStreamName string, sta
 	}
 	return c.svc.GetLogEvents(input)
 }
+
+func isInvalidParameterError(err error) bool {
+	if awsErr, ok := err.(awserr.Error); ok {
+		return awsErr.Code() == constants.InvalidParameterError
```

Comment:
The `cloudwatch` pkg has an `ErrCodeInvalidParameterException` constant we should use for error detection.

Reply:
Oh awesome. Updated!